README.md

```diff
@@ -84,3 +84,16 @@ Host baxter
   HostName baxter.jsk.imi.i.u-tokyo.ac.jp
   User ruser  # password: rethink
 ```
+
+**Setup Creative Camera**
+
+1. Install DepthSenseSDK. You need to login to download it. [Middleware -> Download](http://www.softkinetic.com/)
+
+2. Install ROS package
+
+```
+git clone  https://github.com/ipa320/softkinetic
```

Comment:
Please add `ipa320/softkinetic` in rosinstall.
https://github.com/start-jsk/jsk_apc/blob/master/jsk_2016_01_baxter_apc/rosinstall